### ISSUE TITLE
Documentation updating support for GNU/Linux systems

### ISSUE THREAD

**Issue by n8pjl** [debrouxl/gcc4ti#16]
I just spent the last two hours trying to configure wine to run UpdateInclude.exe to no avail. Any advice and/or patches to make the process work would be wonderful.

On a related note, I have some patches for the docs but will not submit them until I can test them.

**Comment by debrouxl** [comment]
The setup I have on my computer, and which I haven't used for years, largely matches the example configuration I gave in `trunk/tigcc/doc/Programs/HelpSystem.ini`. I provided this example configuration because the setup was a chore to me as well, despite already knowing how to use these documentation tools :)
Some of the symlinks I use point to one level higher than what I documented back in the day, but I don't think that it matters. In ~/.wine/drive_c:
```
doc -> <...>/gcc4ti/trunk/tigcc/doc
docfiles -> <...>/gcc4ti/trunk/tigcc/doc/System
examples -> <...>/gcc4ti/trunk/tigcc/examples
include -> <...>/gcc4ti/trunk/tigcc/include
```

**Comment by n8pjl** [comment]
Yeah, with a bit more digging and experimentation I managed to get  it to work. My biggest issue was that the default configuration in `HelpSystem.ini` (after adjusting the root and following the symlink) points to `<...>/gcc4ti/trunk/tigcc/docfiles/` which does not exist in the source tree.

So my issue had nothing to do with wine, though using wine is still less than convenient.
